pyproject.toml

```diff
@@ -27,6 +27,7 @@ mdx-truly-sane-lists = "^1.2"
 GitPython = "^3.1.20"
 mike = "^1.0.1"
 black = "^21.7b0"
+click = "8.0.4"
```

Comment:
Why was this added? `click` is only a transitive dependency, so I don't see why it should be in `pyproject.toml`.

Reply:
Our pinned version of `black` doesn't run with the latest `click`: https://github.com/psf/black/issues/2964
I tried to update black to the latest 22.3 as someone else suggested, but poetry doesn't seem to find it...

Reply:
That is odd, it has been in use for the last month without problems in the projects which have Dependabot-maintained Python dependencies (e.g., https://github.com/arduino/arduino-lint/pull/365, https://github.com/arduino/library-registry/pull/1243, https://github.com/arduino/library-registry-submission-parser/pull/92, https://github.com/arduino/libraries-repository-engine/pull/113)

Reply:
Ok, I think I found the reason, I did `poetry install` just after changing the dependency, but you must do `poetry update` before. Now everything seems ok... let's wait for the CI to run, and see.